src/core/lib/security/credentials/oauth2/oauth2_credentials.cc

```diff
@@ -450,6 +459,18 @@ void grpc_google_refresh_token_credentials::fetch_oauth2(
   gpr_free(body);
 }
 
+void grpc_google_refresh_token_credentials::add_additional_metadata(
+    grpc_credentials_mdelem_array* md_array) {
+  if (refresh_token_.quota_project_id != nullptr) {
+    grpc_mdelem quota_project_md = GRPC_MDNULL;
```

Comment:
nit: unnecessary line, it can be set to `grpc_mdelem_from_slices`

Reply:
Thanks, done